WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.swift

```diff
@@ -481,10 +480,6 @@ final class ReaderCommentsViewController: UIViewController, WPContentSyncHelperD
         }
 
         WPAnalytics.trackReaderStat(.readerArticleCommentedOn, properties: properties)
-
-        if let railcar {
-            WPAppAnalytics.trackTrainTracksInteraction(.trainTracksInteract, withProperties: railcar)
-        }
```

Comment:
I'm not sure what this "railcar" is, but do we just not care about it anymore?

Reply:
It's something that was added nine years ago and appears to be unused. Posts don't have railcars, and I couldn't find any recent events on Tracks either. I think it's pretty safe to say it's dead code.